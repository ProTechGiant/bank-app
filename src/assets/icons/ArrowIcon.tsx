import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function ArrowIcon({ width = 24, height = 25, color = "#B3B3B3" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M9.70687 6.5L8.29688 7.91L12.8769 12.5L8.29688 17.09L9.70687 18.5L15.7069 12.5L9.70687 6.5Z"
        fill={color}
      />
    </Svg>
  );
}
