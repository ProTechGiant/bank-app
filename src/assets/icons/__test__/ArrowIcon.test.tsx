import { create } from "react-test-renderer";

import { ArrowIcon } from "../ArrowIcon";

describe("icon/ArrowIcon", () => {
  it("renders a ArrowIcon", () => {
    const result = create(<ArrowIcon />);

    expect(result).toMatchSnapshot();
  });
});
