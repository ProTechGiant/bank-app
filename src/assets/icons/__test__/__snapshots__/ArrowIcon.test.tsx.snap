// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Carousel renders a Carousel 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="25"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={25}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 25,
        "width": 24,
      },
    ]
  }
  vbHeight={25}
  vbWidth={24}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M9.70687 6.5L8.29688 7.91L12.8769 12.5L8.29688 17.09L9.70687 18.5L15.7069 12.5L9.70687 6.5Z"
      fill={
        {
          "payload": 4289967027,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`icon/ArrowIcon renders a ArrowIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="25"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={25}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 25,
        "width": 24,
      },
    ]
  }
  vbHeight={25}
  vbWidth={24}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M9.70687 6.5L8.29688 7.91L12.8769 12.5L8.29688 17.09L9.70687 18.5L15.7069 12.5L9.70687 6.5Z"
      fill={
        {
          "payload": 4289967027,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
